cibyl/models/ci/system_factory.py

```diff
@@ -24,6 +24,7 @@ class SystemType(str, Enum):
     """
     JENKINS = 'jenkins'
     ZUUL = 'zuul'
+    ZUULD = 'zuul.d'
```

Comment:
This line can be removed, can't it?

Reply:
Yeah.. that should be removed. Thanks for catch.